rules/app.bzl

```diff
@@ -76,6 +77,9 @@ def ios_application(name, apple_library = apple_library, infoplists_by_build_set
 
     infoplists = select(infoplists_by_build_setting)
 
+    # Keep original default value 'entitlements_validation_mode.loose' if not set
+    entitlements_validation = kwargs.pop("entitlements_validation", entitlements_validation_mode.loose)
```

Comment:
should this instead go in `_IOS_APPLICATION_KWARGS` ?

Reply:
Yep! My bad!